tools/cli/pkg/command/reconciliation_info_test.go

```diff
@@ -93,6 +95,41 @@ func TestReconciliationOperationInfoCommand_Run(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "reconciliation info: happy path - reconciliation found",
+			fields: fields{
+				ctx:    testCtx,
+				output: outputJSON,
+				provideMshipClient: func(url string, _ *http.Client) (mothership.ClientInterface, error) {
+
+					r, err := ioutil.ReadFile("testdata/reconciliationInfoResponse.json")
+					if err != nil {
+						fmt.Print(err)
```

Comment:
extract it to seperate method and add as first arg `testing.T` to abort test in case of error